General/Baselines.md

```diff
@@ -50,6 +50,8 @@ BaselineOf subclass: #BaselineOfMyProject
   package: 'BaselineOfMyProject'
 ``` 
 
+This class should be in a package separated from other package's projects. By convention, we name this package holding the baseline with the same name as the baseline. To summarize, `BaselineOfMyProject` class is in `BaselineOfMyProject` package.
```

Comment:
It's not just a convention I think because if you have a different name Metacello will not be able to resolve it.